integration/tests/bft/gossipsub/topicvalidator/orchestrator.go

```diff
@@ -21,10 +21,10 @@ const (
 	// The numOfAuthorizedEvents allows us to wait for a certain number of authorized messages to be received, this should
 	// give the network enough time to process the unauthorized messages. This ensures us that the unauthorized messages
 	// were indeed dropped and not unprocessed.
-	numOfAuthorizedEvents = 100
```

Comment:
It would be helpful to add warnings / cautions of keeping `numOfAuthorizedEvents` and `numOfUnauthorizedEvents` not too high. Were they reduced to decrease test times? Address flakiness?

Reply:
https://github.com/onflow/flow-go/pull/4652/commits/51c29eab4d7f0021691319aae05151619595eb89